packages/cli/src/commands/generate-block-type-string-union.ts

```diff
@@ -0,0 +1,28 @@
+import { Command } from "commander";
+import { readFile, writeFile } from "fs/promises";
+
+import { type BlockMeta } from "../BlockMeta";
+
+type Options = {
+    inputFile: string;
+    outputFile: string;
+};
+
+export const generateBlockTypeStringUnion = new Command("generate-block-type-string-union")
+    .description("generate block string union from block meta")
+    .option("--input-file <inputFile>", "file to read block meta from", "block-meta.json")
+    .option("--output-file <outputFile>", "file to write block types to", "./src/AvailableBlockTypes.generated.ts")
+    .action(async (options: Options) => {
+        const blockMeta = await readFile(options.inputFile).then((fileContents) => JSON.parse(fileContents.toString()) as BlockMeta[]);
+
+        const uniqueBlockNames = Array.from(new Set(blockMeta.map((block) => block.name)));
+        const sortedBlockMeta = uniqueBlockNames.sort();
+
+        let content = "export type AvailableBlockTypes =";
```

Comment:
Is this really a BlockType? 

At runtime the block types are lowerCase and in the block-meta.json they are PascalCase. 

Are there already different namings for this like BlockType and BlockName? Are they the same?

Reply:
No, it's actual the block names.

> At runtime the block types are lowerCase and in the block-meta.json the names are PascalCase.

This are two different things: the lowerCase variant are the keys of the child blocks in supportedBlocks, the PascalCase are the child blocks' names

I would change the name of this new type to `AllBlocks` (and later `RootBlocks`)